blockchain/vm/interpreter.go

```diff
@@ -123,6 +123,23 @@ func NewEVMInterpreter(evm *EVM) *EVMInterpreter {
 		cfg.JumpTable = jt
 	}
 
+	// Enable the opcode computation cost limit
+	if cfg.ComputationCostLimit == 0 {
+		switch {
+		case evm.chainRules.IsCancun:
+			// approximately 120 ms
+			cfg.ComputationCostLimit = uint64(120000000)
```

Comment:
I have a research project with high gas consumption. Before this PR, transactions had to be split into two to manage computation cost (CC). Now based on this PR, transaction scales up to around 40%. The maximum CC is now achieved in under 70ms on a machine with an Intel i9 processor, 16GB memory, and 20 cores. I presume this value is based on standard CN spec, and my machine appears to consume roughly half of that amount even though the spec is not performant than the CN spec. Could you provide insight into how this value was derived?

Reply:
Please consider the 120ms execution time as a guide time. Depending on the condition of the node, the execution time can vary. For example, i/o load or cpu spec can affect the execution time.